rs/tests/boundary_nodes/integration_test_common/src/lib.rs

```diff
@@ -250,10 +246,10 @@ pub fn api_sync_call_test(env: TestEnv) {
     block_on(async move {
         let cid = install_counter_canister(env.clone(), logger.clone()).await?;
 
-        let transport = ReqwestTransport::create_with_client(format!("https://{host}/"), client)?
-            .with_use_call_v3_endpoint();
-
-        let agent = Agent::builder().with_transport(transport).build()?;
+        let agent = Agent::builder()
+            .with_url(format!("https://{host}/"))
+            .with_http_client(client)
+            .build()?;
         agent.fetch_root_key().await?;
```

Comment:
We lost the `.with_use_call_v3_endpoint();` part here – but I don't know what the replacement is

Reply:
cc @r-birkner

Reply:
Yes, this is fine. The `api/v3/call` endpoint was an option and now it is the default behavior of the agent. So, your change is good.